runtime/calamari/src/weights/cumulus_pallet_xcmp_queue.rs

```diff
@@ -54,13 +54,13 @@ pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> cumulus_pallet_xcmp_queue::WeightInfo for SubstrateWeight<T> {
     // Storage: XcmpQueue QueueConfig (r:1 w:1)
     fn set_config_with_u32() -> Weight {
-        (3_214_000 as Weight)
+        (18_760_000 as Weight)
```

Comment:
XCM seems to have gotten 6x more expensive, might be worth having a look as to why

Reply:
Looks like an outlier. Ran it twice more
run 2 - https://github.com/Manta-Network/Manta/commit/a0e9fe624472c1f93ac614384fe7cbaa3206b1cb#diff-832e23f2f2623e40bfdd2ee080ac8e7bd21e293012315dfadf406d29ec606702
run 3 - https://github.com/Manta-Network/Manta/commit/564cb0ce8033e5e1a928269fd383668c63053b9a#diff-052eb681ebafd55f26e5f95c8ec586004eb56c0ecdc174fbd6e01258db8f86ab

which are much lower and consistent with each other.